docs/smart_rollups.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Smart Rollups
+author: Davis Sawali 
+---
+
+# Smart Optimistic Rollups
+
+Rollups are a permissionless scaling implementation for the Tezos blockchain. The idea is that anyone can originate and operate one or more rollups, increasing the overall throughput of the Tezos blockchain.
+
+In Taquito, we have implemented some of the operations included in Mumbai protocol update in regards to smart rollups. In this document, we will go through the ones that we support. We will also go over smart rollups in a more high level view, if you'd like to understand the feature a bit deeper, you can refer to [this document](https://tezos.gitlab.io/mumbai/smart_rollups.html).
```

Comment:
what do you think about? 
`We will also go over smart rollups for a high level overview,`

Reply:
I don't think 'for' sounds right in this context. I think we can do 'We will also go over smart rollups on a high level,...'

Reply:
I changed it to something a bit more verbose and less vague